tests/starwars/test_dsl.py

```diff
@@ -323,7 +319,7 @@ def test_subscription(ds):
     assert (
         print_ast(query)
         == """subscription {
-  reviewAdded(episode: JEDI) {
+  reviewAdded(episode: 6) {
```

Comment:
question: Is it valid syntax for graphql (using integer for enum)?
In examples graphql enum always used by name without quotas, i.e. `episode: JEDI` looks more correct.

Reply:
Test were not passing with these values, so I fixed them quickly to pass - you need to reconcile what you are passing as input with what you are getting on output in accordance with the `gql`/`graphql-core` behaviour.